lib/tasks/manage.rake

```diff
@@ -156,4 +176,9 @@ namespace :manage do
   task :update_subscriber_list, %i[slug new_title new_slug] => :environment do |_t, args|
     update_subscriber_list(slug: args[:slug], new_title: args[:new_title], new_slug: args[:new_slug])
   end
+
+  desc "Update business finder subscriptions"
+  task update_business_finder_subscriptions: :environment do |_t|
```

Comment:
What do we use `:environment` for here? Does this rake task apply to all environments..?

Reply:
From my understanding it loads the Rails environment and let's me use the existing classes (I'm using SubscriberList, Subscription) and methods